clusterloader2/testing/overrides/kubemark_100_nodes.yaml

```diff
@@ -0,0 +1 @@
+CL2_LOAD_TEST_THROUGHPUT: 20
```

Comment:
There is already one for it - it's this one: https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/overrides/kubemark_load_throughput.yaml